### PR TITLE
Pixels single image downloads

### PR DESCRIPTION
**Pull request by gestchild**
Adds pixel dimensions for single image downloads, i.e. works with a iiif-image location

Screenshot 2020-03-31 at 09 55 28

### CODE REVIEW

**Comment by taceybadgerbrook** [comment]
This should probably be evaluated to show we are supporting user needs.  Is the event this triggers different from the existing view events?

**Comment by gestchild** [comment]
@taceybadgerbrook No the events haven't changed. I'll add a note to GA when this gets deployed

File: catalogue/webapp/components/WorkDetails/WorkDetails.js [hunk 1]
```diff
@@ -74,6 +75,18 @@ const WorkDetails = ({
   itemsRef.current = itemsWithPhysicalLocations;
   const [hasRequestableItems, setHasRequestableItems] = useState(false);
   const singleItem = itemsWithPhysicalLocations.length === 1;
+  const [imageJSON, setImageJSON] = useState(null);
+  const fetchImageJSON = async () => {
+    try {
+      const imageJSON =
+        iiifImageLocation &&
+        (await fetch(iiifImageLocation.url).then(resp => resp.json()));
```
Comment (davidpmccormick) [inline catalogue/webapp/components/WorkDetails/WorkDetails.js]: I always think it's a bit strange mixing `await` and `.then`. Although I guess the double-`await` pattern is weird too.

File: catalogue/webapp/components/WorkDetails/WorkDetails.js [hunk 2]
```diff
@@ -74,6 +75,18 @@ const WorkDetails = ({
   itemsRef.current = itemsWithPhysicalLocations;
   const [hasRequestableItems, setHasRequestableItems] = useState(false);
   const singleItem = itemsWithPhysicalLocations.length === 1;
+  const [imageJSON, setImageJSON] = useState(null);
+  const fetchImageJSON = async () => {
```
Comment (davidpmccormick) [inline catalogue/webapp/components/WorkDetails/WorkDetails.js]: I'd prefer `fetchImageJson`, `imageJson` and `setImageJson` (i.e. not all caps for `JSON`), but I appreciate that that might be getting a bit bike-sheddy!

**Comment by davidpmccormick** [comment]
I still approve.

**Comment by pollecuttn** [comment]
Tacey looking at download usage

**Comment by taceybadgerbrook** [comment]
Usage trends for the period 1-15/4/20 compared to 1/1/20 - 31/3/20 show that this change has resulted inapproximately 100% increased downloads for desktop users.

Here is a link to the PowerPoint slide presentation:
https://wellcomecloud.sharepoint.com/:p:/r/sites/wcdigitalexperience/Shared%20Documents/Analytics/5206%20More%20informative%20single%20image%20download%20buttons.pptx?d=w71855134008a4bb3bb3f787bccd00d13&csf=1&web=1&e=w0DDUB

**Pull request merged**